src/components/Inventory/view/WithdrawalModal.tsx

```diff
@@ -259,82 +259,52 @@ export default function WithdrawalsModal(props: WithdrawalsModalProps): JSX.Elem
                 )}
 
-                {record.batchWithdrawals.map((bw, index) => {
-                  return (
-                    <>
-                      <Grid
-                        item
-                        xs={gridSize()}
-                        sx={marginTop}
-                        paddingRight={paddingSeparator}
-                        key={`batch-not-ready-${index}`}
-                      >
-                        <Textfield
-                          id='notReadyQuantityWithdrawn'
-                          value={bw.notReadyQuantityWithdrawn}
-                          onChange={(id, value) => onChangeQuantity('notReadyQuantityWithdrawn', value)}
-                          type='text'
-                          label={strings.NOT_READY_QUANTITY_REQUIRED}
-                          errorText={validateFields && bw.notReadyQuantityWithdrawn === 0 ? strings.REQUIRED_FIELD : ''}
-                        />
-                      </Grid>
-                      <Grid item xs={gridSize()} sx={marginTop} paddingLeft={paddingSeparator}>
-                        <DatePicker
-                          id='readyByDate'
-                          label={strings.ESTIMATED_READY_DATE}
-                          aria-label={strings.ESTIMATED_READY_DATE}
-                          value={record.readyByDate}
-                          onChange={changeDate}
-                          errorText={validateFields && !record.withdrawnDate ? strings.REQUIRED_FIELD : ''}
-                        />
-                      </Grid>
-                      <Grid
-                        item
-                        xs={gridSize()}
-                        sx={marginTop}
-                        paddingRight={paddingSeparator}
-                        key={`batch-ready-${index}`}
-                      >
-                        <Textfield
-                          id='readyQuantityWithdrawn'
-                          value={bw.readyQuantityWithdrawn}
-                          onChange={(id, value) => onChangeQuantity('readyQuantityWithdrawn', value)}
-                          type='text'
-                          label={strings.READY_QUANTITY_REQUIRED}
-                          errorText={validateFields && bw.readyQuantityWithdrawn === 0 ? strings.REQUIRED_FIELD : ''}
-                        />
-                      </Grid>
-                      <Grid item xs={gridSize()} sx={marginTop} paddingLeft={paddingSeparator}>
-                        <Textfield
-                          id='withdrawnQuantity'
-                          value={withdrawQuantity}
-                          onChange={onChange}
-                          type='text'
-                          label={strings.WITHDRAW_QUANTITY}
-                          display={true}
-                        />
-                      </Grid>
-                    </>
-                  );
-                })}
+                {record.batchWithdrawals.map((bw, index) => (
```

Comment:
i mostly moved grid content up to fill previous deleted grid-content (Ready by date).
Also fixed a non-unique key error (introduced Grid container)